### ISSUE TITLE
Does dependabot scan kubernetes manifest file and helm charts to detect deprecated api-versions?

### ISSUE THREAD

**Issue by SethMuskan** [dependabot/dependabot-core#7221]
Hi Team,

Does Dependabot scan kubernetes manifest file and helm charts and report if there is any deprecated api versions?

I have below kubernetes manifest file present in my repo and I want dependabot to create a PR to update apiversion to v1.

"deploy.yaml"
apiVersion: apps/v1beta2
kind: Deployment

Please suggest if i need to initiate discussion under some other forum.

**Comment by jeffwidman** [comment]
No, and also I'm not sure this would be a great fit for Dependabot... 

API version changes like that are often breaking changes meaning that any PR's we create are likely to be broken due to needing other code changes that almost always require human intervention to get right. That's kinda the whole point of versioning APIs.

I do see Renovate supports this, which surprised me, are the PR's they're creating for these API changes typically broken?

Or am I misunderstanding k8s API versioning and most of the time the API version can safely be bumped w/o breaking stuff?
